build.gradle

```diff
@@ -638,6 +638,8 @@ project(':core') {
   dependencies {
     compile project(':clients')
     compile libs.jacksonDatabind
+    compile libs.jacksonModuleScala
+    compile libs.jacksonDataformatCsv
```

Comment:
@vahidhashemian I don't think we should have this dependency. We have avoided the Jackson module for Scala on purpose as it doesn't provide enough value. It would be good to fix this before 2.3.0 if at all possible. cc @hachikuji

Reply:
To clarify, I'm referring to the Scala module (I got the line slightly wrong). The CSV one is probably fine.

Reply:
How this was isolated:
-  comment out **libs.jacksonModuleScala** dependency in build.gradle
- execute **./gradlew -PscalaVersion=2.12 jar**

Result: two compile errors in `kafka.admin.ConsumerGroupCommand` (**_core_** module)

I volunteer to help with this.

Reply:
@dejan2609 If you submit a PR, I'll help review it.

Reply:
@ijuma Thanks for catching this. I wasn't aware of the limitation.
@dejan2609 Thanks for volunteering. If you like, you can open a JIRA for easier tracking of the issue.

Reply:
@vahidhashemian Sure, I will open JIRA ticket.

@Ijuma FYI: as a former Java developer that switched to build/release engineering I can't deliver quickly (but I assume this is not a post-haste).

Reply:
JIRA ticket is created here: https://issues.apache.org/jira/browse/KAFKA-8466 **_Remove 'jackson-module-scala' dependency (and replace it with some code)_**